coderd/notifications/metrics_test.go

```diff
@@ -263,9 +263,10 @@ func TestPendingUpdatesMetric(t *testing.T) {
 	interceptor.proceed.Broadcast()
 
 	// Validate that the store synced the expected number of updates.
-	require.Eventually(t, func() bool {
+	require.Eventuallyf(t, func() bool {
 		return syncer.sent.Load() == 1 && syncer.failed.Load() == 1
-	}, testutil.WaitShort, testutil.IntervalFast)
+	}, testutil.WaitLong, testutil.IntervalFast,
+		"sent: %d, failed: %d", syncer.sent.Load(), syncer.failed.Load())
```

Comment:
Drive-by: this test flaked once but I couldn't find the reason after a quick look, so I'm extending the wait time a bit and adding some debugging.